map.py

```diff
@@ -91,30 +101,38 @@ def execute_user_move(user, new_column, new_row):
     user["column"] = new_column
     user["row"] = new_row
 
-def is_coordinate_passible(map, column, row):
-    if(map[column][row] == HH or map[column][row] == VV):
-        return False
-    elif(map[column][row] == EE):
+def is_coordinate_passable(map, column, row):
+    if(map[row][column] == EE):
         return True
+    else:
+        return False
 
 def can_player_move_to_coordinate(map, column, row):
     if(is_coordinate_on_map(map, column, row) == False): return False
-    if(is_coordinate_passible(map, column, row) == False): return False
+    if(is_coordinate_passable(map, column, row) == False): return False
 
     return True
 
 def game_board():
```

Comment:
This "main loop" does not belong in the map. it belongs, at least for right now, in your script entry, which is adventure.py (or similar). This issue is likely why we have the concept of user leaking a bit.